xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -597,7 +597,7 @@ void CGUIDialogVideoInfo::Play(bool resume)
     return;
   }
 
-  CFileItem movie(*m_movieItem->GetVideoInfoTag());
+  CFileItem movie(*m_movieItem);
```

Comment:
So the original idea was to add
if (!item.m_bIsFolder && !item.IsPVR() **|| item.IsPlugin()**)
to
xbmc/video/windows/GUIWindowVideoBase.cpp 
See above in the hidden change request, it was requested that I dont do that there and find the root cause. 
I found the root cause as the IsFolder not being set so then is when I realised the IsFolder was not being set when using the VideoInfoTag constructor.
If you think it is better to go back to the original idea of adding || item.IsPlugin() then I can do that.